### DISCUSSION TITLE
reset() clears input field instead of showing default

### DISCUSSION THREAD

**Discussion by vnugent** [react-hook-form/react-hook-form#10118] [Q&A] [answered]
I have a first name text field initialized to "Jane Doe" and it correctly displays "Jane Doe" as expected.  However, when I hit the Reset button the field name is blank.  I expect `reset()` to apply whatever the defaultValues object is but it's not the case here. 

https://codesandbox.io/s/rhf-reset-hk8hxh?file=/src/App.tsx:273-300

**Comment by Sajarin-M** [comment, marked as answer]
That is because you are using `type="reset"` on the button. Use `type="button"` instead.

**Comment by vnugent** [comment]
This is it! Thank you @Sajarin-M
